### PR TITLE
[ja] Replace deprecated fullversion shortcode

### PR DESCRIPTION
**Pull request by rolfedh**
fixes https://github.com/kubernetes/website/issues/41486
Related to https://github.com/kubernetes/website/issues/41005

### CODE REVIEW

**Comment by rolfedh** [comment]
cc: @seokho-son @sftim

**Comment by atoato88** [comment]
@rolfedh 
There are some comment from other reviewer, could you update the contents?

File: content/ja/docs/tasks/administer-cluster/kubeadm/upgrading-windows-nodes.md [hunk 1]
```diff
@@ -83,10 +83,10 @@ Windowsノードをアップグレードする前にコントロールプレー
 ### kube-proxyをアップグレード
 
 1. Kubernetes APIにアクセスできるマシンから、次を実行します、
-もう一度{{< param "fullversion" >}}を目的のバージョンに置き換えます:
+もう一度{{< skew currentPatchVersion >}}を目的のバージョンに置き換えます:
 
     ```shell
-    curl -L https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/kube-proxy.yml | sed 's/VERSION/{{< param "fullversion" >}}/g' | kubectl apply -f -
+    curl -L https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/kube-proxy.yml | sed 's/VERSION/{{< skew currentPatchVersion >}}/g' | kubectl apply -f -
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply content/ja/docs/tasks/administer-cluster/kubeadm/upgrading-windows-nodes.md:89]: Updated. Thanks.

File: content/ja/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md [hunk 2]
```diff
@@ -104,11 +104,11 @@ LinuxベースのKubernetesコントロールプレーンノードを取得し
 
     これで、Windows互換バージョンのFlannelおよびkube-proxyを追加できます。
     互換性のあるバージョンのkube-proxyを確実に入手するには、イメージのタグを置換する必要があります。
-    次の例は、Kubernetes{{< param "fullversion" >}}の使用方法を示していますが、
+    次の例は、Kubernetes{{< skew currentPatchVersion >}}の使用方法を示していますが、
     独自のデプロイに合わせてバージョンを調整する必要があります。
 
     ```bash
-    curl -L https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/kube-proxy.yml | sed 's/VERSION/{{< param "fullversion" >}}/g' | kubectl apply -f -
+    curl -L https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/kube-proxy.yml | sed 's/VERSION/{{< skew currentPatchVersion >}}/g' | kubectl apply -f -
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply content/ja/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md:111]: Updated. Thanks.

**Comment by seokho-son** [comment]
Hi @kubernetes/sig-docs-ja-reviews and @kubernetes/sig-docs-ja-owners 
I am SIG-Docs localization subproject co-lead, and hope to handle this PR quickly to finalize https://github.com/kubernetes/website/issues/41005 .

PTAL :)

File: content/ja/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md [hunk 3]
```diff
@@ -104,11 +104,11 @@ LinuxベースのKubernetesコントロールプレーンノードを取得し
 
     これで、Windows互換バージョンのFlannelおよびkube-proxyを追加できます。
     互換性のあるバージョンのkube-proxyを確実に入手するには、イメージのタグを置換する必要があります。
-    次の例は、Kubernetes{{< param "fullversion" >}}の使用方法を示していますが、
+    次の例は、Kubernetes{{< skew currentPatchVersion >}}の使用方法を示していますが、
```
Comment (sftim) [inline content/ja/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md:107]: (nit)
```suggestion
    次の例は、Kubernetes {{< skew currentPatchVersion >}}の使用方法を示していますが、
```

**Review by sftim — COMMENTED** [review]
LGTM

**Comment by atoato88** [comment]
Almost LGTM.
Could you update the content according to the comment above?

**Comment by sftim** [comment]
@atoato88 I'm happy to see this land even with the nit. We can clean up that last detail in a fixup PR.

**Comment by atoato88** [comment]
@nasa9084 
Could you review this?

File: content/ja/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md (hunk 3 above)
Comment (rolfedh) [reply content/ja/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md]: Updated.

**Comment by sftim** [comment]
Relevant changes since https://github.com/kubernetes/website/pull/41527#issuecomment-1631431739 are trivial.

/lgtm

**Comment by nasa9084** [comment]
sorry for late reviewing
/approve

**Pull request merged**